src/status_im/transport/inbox.cljs

```diff
@@ -177,20 +177,19 @@
  :inbox/check-peer-added
   ;; We check if the wnode is part of the peers list
   ;; if not we dispatch a new fetch-peer event for later
- (fn [{:keys [db]} [_ peers retries]]
-   (let [web3     (:web3 db)
-         wnode    (get-current-wnode-address db)]
+ (fn [{{:keys [web3 network-status] :as db} :db} [_ peers retries]]
+   (let [wnode    (get-current-wnode-address db)]
      (log/info "offline inbox: fetch-peers response" peers)
      (if (registered-peer? peers wnode)
        {::mark-trusted-peer {:web3  web3
                              :wnode wnode}}
        (do
          (log/info "Peer" wnode "is not registered. Retrying fetch peers.")
          (let [delay (if (< retries 3) 300 5000)]
```

Comment:
would be great to extract all magic values to `def`s

Reply:
Extracted